### PR TITLE
initial travis-CI support

### PR DESCRIPTION
**Pull request by raveit65**

### CODE REVIEW

**Comment by raveit65** [comment]
@monsta 
Are debian and ubuntu build requires the same?
Which debian version provide docker images?
Than i will add debian to config.
I forked m-n-d an did a successful test.
https://github.com/raveit65/mate-notification-daemon/pull/3

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sudo: required
+services:
+  - docker
+
+before_install:
+  - curl -L -o docker-build https://raw.githubusercontent.com/mate-desktop/mate-dev-scripts/travis/travis/docker-build
```
Comment (yetist) [inline .travis.yml]: After merge that PR, the url here should be changed to https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/docker-build.

Comment (raveit65) [reply .travis.yml]: done.

**Comment by yetist** [comment]
Yes, if use debian the cmd is not setup.

**Comment by raveit65** [comment]
Ok, than i will merge PR like it is and we need to add debian to docker-build script first.

**Comment by raveit65** [comment]
\o/ it works.
https://travis-ci.org/mate-desktop/mate-notification-daemon/builds/484804313

**Pull request merged**